packages/cfd/src/sass/cfd.scss

```diff
@@ -191,6 +191,10 @@
         .dc-text {
             font-size: var(--text-size-xxs);
             text-align: var(--text-align-left);
+            & > a {
+                font-weight: bold;
+                text-transform: lowercase;
```

Comment:
We're making `terms and conditions` lowercase just for zero spread which is bit weird TBH.